### PR TITLE
Migrate zipformer model to other Chinese datasets

### PR DESCRIPTION
**Pull request by JinZr**
Provided datasets are listed below: 
1. aidatatng_200zh
2. aishell
3. aishell2
4. aishell4

### CODE REVIEW

File: egs/aishell/ASR/RESULTS.md [hunk 1]
```diff
@@ -1,6 +1,149 @@
...
+
+#### Zipformer (Non-streaming)
+
+[./zipformer](./zipformer)
+
+It's reworked Zipformer with Pruned RNNT loss.
+**Caution**: It uses `--context-size=1`.
+
+##### normal-scaled model, number of model parameters: 73412551, i.e., 73.41 M
+
```
Comment (csukuangfj) [inline egs/aishell/ASR/RESULTS.md:13]: Could you please upload pre-trained models to huggingface and post a link here?

File: egs/aishell2/ASR/RESULTS.md [hunk 2]
```diff
@@ -1,8 +1,51 @@
 ## Results
 
-### Aishell2 char-based training results (Pruned Transducer 5)
+### Aishell2 char-based training results 
 
-#### 2022-07-11
+#### Zipformer
+
+[./zipformer](./zipformer)
```
Comment (csukuangfj) [inline egs/aishell2/ASR/RESULTS.md:7]: Could you upload the models to huggingface?

File: egs/aishell4/ASR/RESULTS.md [hunk 3]
```diff
@@ -2,6 +2,50 @@
...
+#### Zipformer
+
+[./zipformer](./zipformer)
+
+It's reworked Zipformer with Pruned RNNT loss, note that results below are produced by model trained on data without speed perturbation applied.
+
+**⚠️ If you prefer to have the speed perturbation disabled, please pass `false` to `--perturb-speed` of the `prepare.sh` script as demonstrated below.**
+
+|                        | test | comment                               |
+|------------------------|------|---------------------------------------|
```
Comment (csukuangfj) [inline egs/aishell4/ASR/RESULTS.md:16]: Could you upload the models to huggingface?

**Review by csukuangfj — CHANGES_REQUESTED** [review]
Thanks! Left some minor comments about uploading pre-trained models to huggingface.

File: egs/aidatatang_200zh/ASR/prepare.sh [hunk 4]
```diff
@@ -7,6 +7,8 @@ set -eou pipefail
 
 stage=-1
 stop_stage=100
+perturb_speed=true
```
Comment (csukuangfj) [inline egs/aidatatang_200zh/ASR/prepare.sh:10]: Could you update RESULTS.md for aidatatang_200zh about zipformer?

**Review by csukuangfj — COMMENTED** [review]
Thanks!

File: egs/aishell/ASR/README.md [hunk 5]
```diff
@@ -1,10 +1,12 @@
 
 # Introduction
 
-Please refer to <https://icefall.readthedocs.io/en/latest/recipes/Non-streaming-ASR/aishell/index.html>
-for how to run models in this recipe.
+Please refer to <https://icefall.readthedocs.io/en/latest/recipes/Non-streaming-ASR/aishell/index.html> for how to run models in this recipe.
```
[the remark this answers is not in the document]
Comment (JinZr) [reply egs/aishell/ASR/README.md]: the latest commit should fix this issue, thanks!

**Review by csukuangfj — APPROVED** [review]

**Pull request merged**